### PR TITLE
Update batch-cohort.md

### PR DESCRIPTION
**Pull request by Leoaqr**
- Includes description and setup for the new "Batch (Profile)" Cohort destination 
- Updates related to the latest version of the "Batch" Cohort destination

# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Review by Brandoncharleskhoo1 — APPROVED** [review]

File: docs/data/destinations/batch-cohort.md [hunk 1]
```diff
@@ -15,23 +15,29 @@ description: Sync cohorts from Amplitude to Batch
 - You must enable this integration in each Amplitude project you want to use it in.
 - This integration requires a paid Batch plan.
 - The integration requires both custom audiences and third-party connections in Batch.
-- Batch expects an identifier that matches the Batch Custom User ID field. This means the `user_id` or user property you select in Amplitude must contain the same identifier as the one tracked in Batch.
+- Batch expects an identifier that matches the Batch Custom User ID or the Installation ID field. This means the `user_id` or user property you select in Amplitude must contain the same identifier as the one tracked in Batch in one of these fields.
 
 ## Setup
+Two types of cohort destinations can be setup for Batch:
```
Comment (markzegarelli) [inline docs/data/destinations/batch-cohort.md:21]:
```suggestion

Batch supports two types of cohort destination:
```

File: docs/data/destinations/batch-cohort.md [hunk 2]
```diff
@@ -15,23 +15,29 @@ description: Sync cohorts from Amplitude to Batch
...
+
+- Use the **Batch (Profile)** destination if you want your cohorts to be synced with your Batch Omnichannel project.
```
Comment (markzegarelli) [inline docs/data/destinations/batch-cohort.md:23]:
```suggestion
- Use the **Batch (Profile)** destination to sync your cohorts with your Batch Omnichannel project.
```

File: docs/data/destinations/batch-cohort.md [hunk 3]
```diff
@@ -15,23 +15,29 @@ description: Sync cohorts from Amplitude to Batch
...
+- Use the **Batch** destination if you want your cohorts to be synced with a specific mobile app or website.
```
Comment (markzegarelli) [inline docs/data/destinations/batch-cohort.md:24]:
```suggestion
- Use the **Batch** destination to sync your cohorts with a specific mobile app or website.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Comment by markzegarelli** [comment]
Not sure why this merged....Did not intend for it to do so.

**Pull request merged**